File: BackEnd/Models/Dictionary/Invoice/Invoice.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models
{
    public class Invoice : BaseModel
    {
        public Guid InvoiceID { get; set; }
        public Guid ShopID { get; set; }
        public string InvoiceNo { get; set; }
        public string StaffCode { get; set; }
        public string StaffName { get; set; }
        public string CustomerPhone { get; set; }
        public bool IsCash { get; set; }
        public string Description { get; set; }
        public string CustomerAddress { get; set; }
        public string CustomerName { get; set; }
        public DateTime InvoiceDate { get; set; }
        public decimal TotalMoney { get; set; }
    }
}

File: FrontEnd/MshopUI/src/app/models/invoice/invoice.ts
import { BaseModel } from '../base-model/base-model';

export class Invoice extends BaseModel {
    InvoiceID: string;
    ShopID: string;
    InvoiceNo: string;
    StaffCode: string;
    StaffName: string;
    CustomerPhone: string;
    IsCash: boolean = true;
    Description: number;
    CustomerAddress: string;
    CustomerName: string;
    InvoiceDate?: Date = new Date();
    TotalMoney: number = 0;
}